src/main/java/crawlercommons/robots/SimpleRobotRulesParser.java

```diff
@@ -17,11 +17,7 @@
 package crawlercommons.robots;
 
 import java.io.IOException;
-import java.net.HttpURLConnection;
-import java.net.MalformedURLException;
-import java.net.URL;
-import java.net.URLConnection;
-import java.net.URLDecoder;
+import java.net.*;
```

Comment:
Do we have a policy around wildcard imports? Other projects I work say to avoid.